app/client/views/confirmation/confirmation.html

```diff
@@ -365,20 +366,18 @@
             <p>
               <a
                 target="_blank"
-                href="https://go.hackmit.org/code-of-conduct">
-                Hacker Code of Conduct
+                href="https://static.mlh.io/docs/mlh-code-of-conduct.pdf">
+                MLH Code of Conduct
               </a>
             </p>
             <p>
               It is extremely important to us that attendees follow a set of guidelines to make the event fun for everyone involved. Please read through and make sure you understand what we will be expecting of you.
             </p>
-            <p>
-              By writing my name below, I affirm that I have read and will abide by the code of conduct detailed above.
-            </p>
-            <input type="text"
-                   name="signatureCodeOfConduct"
-                   placeholder="Your Full Legal Name"
-                   ng-model="user.confirmation.signatureCodeOfConduct">
+            <div class="ui checkbox">
+              <input id="agreeCodeOfConduct"
+              type="checkbox" ng-model="user.confirmation.agreeCodeOfConduct">
```

Comment:
You should check the User model to make sure that this is stored properly. Also, if you can find all instances of signatureCodeOfConduct and make sure nothing depends on it that'd be great

Reply:
Ignore the part about the user model (sorry I'm on mobile and it won't let me edit the comment).

Reply:
I checked and those are the only 2 instances. I tested it and it seems ok

Reply:
👍